helix-term/src/ui/document.rs

```diff
@@ -313,6 +315,7 @@ pub struct TextRenderer<'a> {
     pub nbsp: String,
     pub space: String,
     pub tab: String,
+    pub virtual_tab: String,
```

Comment:
Must this be `String`? Can it be integer instead?

Reply:
No we need the allocation to be able to hand out `&str` to it, just like the `tab` field. The tab width is not the interesting part here, we already know that from the grapheme.